evm_loader/program/src/instruction.rs

```diff
@@ -250,6 +252,9 @@ impl<'a> EvmInstruction<'a> {
                 let step_count = step_count.try_into().ok().map(u64::from_le_bytes).ok_or(InvalidInstructionData)?;
                 EvmInstruction::Continue {step_count}
             },
+            11 => {
```

Comment:
11 code already used in #167 for iterative execution frou account data (contract deploy). Please, use 12 value.